.github/workflows/test.yml

```diff
@@ -8,17 +8,15 @@ jobs:
     strategy:
       matrix:
         version:
-          # - 2.3 # Debian Stretch
-          # - 2.5 # Debian Buster
-           - 2.7 # Debian Bullseye
-          # - 3.0 # Debian Bookworm
+          - '2.7'
+          - '3.0'
     steps:
       - uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.version }}
 
       - name: Check out repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Out of curiosity why does this need to be updated?

Reply:
Because `actions/checkout@v2` uses Node.js 12, which is deprecated in GitHub actions. Running this action causes a deprecation warning to appear.

![image](https://user-images.githubusercontent.com/31414818/220209141-18e2334c-15bd-4def-800d-d6028234df64.png)

Action Run with deprecation warnings: https://github.com/Verseth/aliquot/actions/runs/4186751593

Blog post by GitHub: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/